src/test/kotlin/logic/use_cases/project/CreateNewProjectUseCaseTest.kt

```diff
@@ -7,23 +7,27 @@ import io.kotest.matchers.shouldBe
 import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Test
 import org.qudus.squad.logic.exceptions.AccessDeniedException
-import org.qudus.squad.logic.exceptions.InvalidProjectInfo
+import org.qudus.squad.logic.exceptions.InvalidProjectDescriptionException
+import org.qudus.squad.logic.exceptions.InvalidProjectTitleException
+import org.qudus.squad.logic.validation.ProjectDataValidationUseCase
 import org.qudus.squad.model.entity.User
 import org.qudus.squad.model.entity.UserRole
 
 class CreateNewProjectUseCaseTest {
 
     private lateinit var fakeProjectRepository: FakeProjectRepository
     private lateinit var createNewProjectUseCase: CreateNewProjectUseCase
+    private lateinit var projectDataValidationUseCase: ProjectDataValidationUseCase
 
     @BeforeEach
     fun setup() {
         fakeProjectRepository = FakeProjectRepository()
-        createNewProjectUseCase = CreateNewProjectUseCase(fakeProjectRepository)
+        projectDataValidationUseCase = ProjectDataValidationUseCase()
+        createNewProjectUseCase = CreateNewProjectUseCase(fakeProjectRepository, projectDataValidationUseCase)
     }
 
     @Test
-    fun `createNewProject should be allowed for admin to create project successfully then return the created project`() {
+    fun `should return the the created project when admin create project seccessfully`() {
```

Comment:
(the) duplicate twice
```suggestion
    fun `should return the created project when admin create project seccessfully`() {
```

Reply:
اصلا